Libraries/D3D12RaytracingFallback/src/TraverseFunction.hlsli

```diff
@@ -606,7 +606,11 @@ bool Traverse(
                     else // if it's a bottom level
                     {
                         MARK(8, 0);
-                        bool geomOpaque = true; // TODO: This should be looked up with the triangle data.
+
```

Comment:
Lets leave this off for a separate change

Reply:
Whoops, snuck in there.